lib/lndclient/errors.ts

```diff
@@ -2,14 +2,20 @@ import errorCodesPrefix from '../constants/errorCodesPrefix';
 
 const codesPrefix = errorCodesPrefix.LND;
 const errorCodes = {
-  LND_IS_DISABLED: codesPrefix.concat('.1'),
+  LND_IS_DISCONNECTED: codesPrefix.concat('.1'),
+  LND_IS_NOT_CONFIGURED: codesPrefix.concat('.2'),
```

Comment:
Here too I actually prefer "disabled" over "not configured" which I think is more clear.

Reply:
done